jupyterhub/values.yaml

```diff
@@ -3,6 +3,9 @@
 fullnameOverride: ""
 nameOverride:
 
+# enable or disable jupyterhub if it is included as a dependency in a parent helm chart
+enabled:
```

Comment:
What do other charts typically do? Do they leave this null, or default to `true`?

Reply:
I suggest its made null as default here, true/false has no meaning by this chart anyhow so making it null seems less confusing, and both false/true and can be defaults for a chart depending on this - such chart should define enabled to false/true explicitly.

Reply:
Most charts dont need an entry for this, but we have a strict schema.